builtins/amp-ad.js

```diff
@@ -49,6 +49,11 @@ export function installAd(win) {
 
   class AmpAd extends BaseElement {
 
+    /** @override */
+    getElementPriority() {
```

Comment:
+1 for `getPriority`. Maybe a better contextual name might be better. Something like `getRenderPriority` - depends on what priority is used for in resources. Though I am ok with `getPriority`